packages/cli/src/commands/dbCommands/studio.js

```diff
@@ -1,32 +1,26 @@
 import path from 'path'
 import fs from 'fs'
 
-import terminalLink from 'terminal-link'
-
 import { runCommandTask, getPaths } from 'src/lib'
+import { schema, epilogue } from 'src/commands/dbCommands/options'
```

Comment:
Maybe rename `schema` to `schemeOpt`, or do something like:

`import * as options from 'src/commands/dbCommands/options'`

Reply:
Fixed in https://github.com/redwoodjs/redwood/pull/1424/commits/180c1fe80ae45835865c830b9515cffba05ce45c.